apps/mobile/.agents/skills/building-native-ui/references/tabs.md

```
@@ -0,0 +1,417 @@
+			<Stack.Screen name="index" options={{ title: 'Home', headerLargeTitle: true }} />
+			<Stack.Screen name="details" options={{ title: 'Details' }} />
+		</Stack>
+	);
+}
+```
+
+## Custom Web Layout
+
+Use platform-specific files for separate native and web tab layouts:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)